# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/wr/WebRenderImageHost.h
# Commit: 31935f164ef0
# Full Hash: 31935f164ef0d6c4668f211482c139847dcab7f2
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2023-12-13 00:43:26
# Regressor Bug: 1829026
# File Overlap Count: 1
# Description:
#   Bug 1829026 - Update CanvasTranslator to work with DrawTargetWebgl. r=aosmond
#   
#   This adds the necessary infrastructure for CanvasTranslator to allocate DrawTargetWebgl
#   instead of just allocating TextureData, and to use RemoteTextureMap to handle sending
#   the DrawTargetWebgl frames to the compositor.
# ==============================================================================

diff -r 13c9bf69085f -r 31935f164ef0 gfx/layers/wr/WebRenderImageHost.h
--- a/gfx/layers/wr/WebRenderImageHost.h	Tue Dec 12 07:35:02 2023 +0000
+++ b/gfx/layers/wr/WebRenderImageHost.h	Tue Dec 12 07:35:03 2023 +0000
@@ -88,6 +88,7 @@
 
   std::deque<CompositableTextureHostRef> mPendingRemoteTextureWrappers;
   bool mWaitingReadyCallback = false;
+  bool mWaitForRemoteTextureOwner = true;
 
   Maybe<RemoteTextureOwnerId> mRemoteTextureOwnerIdOfPushCallback;
   base::ProcessId mForPidOfPushCallback;